docs/tutorial/extensions.md

```diff
@@ -45,6 +45,49 @@ The wrapped function:
 If any of these constraints are violated, the outcome is undefined.
 {% endhint %}
 
+## fhe.multivariate(function)
+
+Allows you to wrap any multivariate function into a table lookup:
+
+```python
+import numpy as np
+from concrete import fhe
+
+def keep(value, keep):
+    return value if keep else np.zeros_like(value, dtype=np.int64)
+
+def multivariate_function(x, y):
```

Comment:
keep same naming value, make_it_zero/keep

Reply:
I've changed the signature to:
```python
def value_if_condition_else_zero(value, condition):
    pass
```